### PR TITLE
Modify Actor type with details

### PR DESCRIPTION
**Pull request by DeepanshuA**
# Description

Modify Internal Actor Types, getting used in Workflows and Activities.

## Issue reference

Please reference the issue this PR is linked to: #6013 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [x] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs https://github.com/dapr/docs/issues/3346
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: pkg/runtime/wfengine/wfengine.go [hunk 1]
```diff
@@ -49,15 +49,32 @@ type WorkflowEngine struct {
...
+}
+
 func IsWorkflowRequest(path string) bool {
 	return backend.IsDurableTaskGrpcRequest(path)
 }
 
-func NewWorkflowEngine() *WorkflowEngine {
+func NewWorkflowEngine(config *WFConfig) *WorkflowEngine {
+	WorkflowActorType = WorkflowActorType + "." + config.AppID + "." + getNamespace()
+	ActivityActorType = ActivityActorType + "." + config.AppID + "." + getNamespace()
```
Comment (cgillum) [inline pkg/runtime/wfengine/wfengine.go:77]: nit: namespace should come before the app ID.

File: pkg/runtime/wfengine/wfengine.go [hunk 2]
```diff
@@ -49,15 +49,32 @@ type WorkflowEngine struct {
...
 	errExecutionAborted = errors.New("execution aborted")
 )
 
+// WFConfig is the configuration for the workflow engine
+type WFConfig struct {
+	AppID string
+}
+
+func getNamespace() string {
+	namespace := os.Getenv("NAMESPACE")
```
[the remark this answers is not in the document]
Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go:64]: I tried to look for ways to fetch namespace, and it seems we are doing similar at some other places too in runtime: https://github.com/dapr/dapr/blob/e988ea9b92f715a59aa3e8bacba8863b97ef4ce6/pkg/runtime/runtime.go#L356

File: pkg/runtime/wfengine/wfengine.go (hunk 1 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go]: Done

File: pkg/runtime/wfengine/wfengine.go [hunk 3]
```diff
@@ -49,15 +49,32 @@ type WorkflowEngine struct {
...
+}
+
 func IsWorkflowRequest(path string) bool {
 	return backend.IsDurableTaskGrpcRequest(path)
 }
 
-func NewWorkflowEngine() *WorkflowEngine {
+func NewWorkflowEngine(config *WFConfig) *WorkflowEngine {
+	WorkflowActorType = WorkflowActorType + "." + getNamespace() + "." + config.AppID
+	ActivityActorType = ActivityActorType + "." + getNamespace() + "." + config.AppID
```
Comment (mukundansundar) [inline pkg/runtime/wfengine/wfengine.go:77]: ideally should be `ns.appID.<var>` namespace should be a prefix?

Comment (mukundansundar) [reply pkg/runtime/wfengine/wfengine.go:77]: Please add UTs for the same...

File: pkg/runtime/wfengine/wfengine_test.go [hunk 4]
```diff
@@ -760,7 +760,9 @@ func startEngine(ctx context.Context, r *task.TaskRegistry) (backend.TaskHubClie
 }
 
 func getEngine() *wfengine.WorkflowEngine {
-	engine := wfengine.NewWorkflowEngine()
+	engine := wfengine.NewWorkflowEngine(&wfengine.WFConfig{
+		AppID: "test",
+	})
```
Comment (mukundansundar) [inline pkg/runtime/wfengine/wfengine_test.go:765]: in the line below https://github.com/dapr/dapr/pull/6192/files#diff-acb2ead1c43373f11db66d9821d6bfa8b1e16ee5566bc89fb0936cedefcd4019R781
Please do not panic, fail the test after planning in `t` to this method.

File: pkg/runtime/wfengine/wfengine.go (hunk 3 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go:77]: > ideally should be `ns.appID.<var>`. `namespace` should be a prefix?

Actually, the purpose is just to differentiate two Apps and if namespace available, then specifying that too, for better differentiation and recognition by underlying WF engine. 
Right now, many entries are formed in state Stores for a Workflow and Activities, with Internal Actor having representation like:

dapr.internal.wfengine.workflow.default.wfapp

dapr.internal.wfengine.activity.default.wfapp

I feel it would just elongate the Actor type naming, with `namespace` and `appId`, without serving any actual purpose.
WDYT?

Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go:77]: > Please add UTs for the same...

Now, asserting WorkflowActorType and ActivityActorType in `TestStartWorkflowEngine`.

File: pkg/runtime/wfengine/wfengine.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (cgillum) [reply pkg/runtime/wfengine/wfengine.go:62]: Instead of copying that logic, let's instead make a call into the existing API to referenced to get the namespace. That way if they update it to support more than just environment variables, our logic will continue to be consistent with the rest of the system.

File: pkg/runtime/wfengine/wfengine.go [hunk 5]
```diff
@@ -49,15 +47,32 @@ type WorkflowEngine struct {
 }
 
 var (
+	WorkflowActorType   = ""
+	ActivityActorType   = ""
```
Comment (cgillum) [inline pkg/runtime/wfengine/wfengine.go:51]: I don't feel great about having global variables that are mutable. Can we instead make these fields (or getter methods, like `func (wfe *WorkflowEngine) GetWorkflowActorType() string`) on the `WorkflowEngine` struct? I see that we initialize these in `NewWorkflowEngine` so I assume it would be easy to make this change. Or if we don't even need these at all, we can just remove them.

File: pkg/runtime/wfengine/wfengine_test.go [hunk 6]
```diff
@@ -779,7 +783,7 @@ func getEngine() *wfengine.WorkflowEngine {
 	})
 
 	if err := actors.Init(); err != nil {
-		panic(err)
+		assert.FailNow(t, "failed to init actors: %s", err)
```
Comment (cgillum) [inline pkg/runtime/wfengine/wfengine_test.go:786]: It's better to use `assert.NoError(t, err)` for cases like this.

**Review by cgillum — COMMENTED** [review]
This is looking good - just a few small things.

File: pkg/runtime/wfengine/wfengine.go (hunk 3 above)
Comment (cgillum) [reply pkg/runtime/wfengine/wfengine.go]: @DeepanshuA I think Mukandan is just suggesting a rearrangement of the existing components, and not asking that we literally add "namespace" and "appId". How about if we want to rearrange to something like `dapr.internal.mynamespace.myapp.workflow` and `dapr.internal.mynamespace.myapp.activity`? That might make for a slightly cleaner hierarchy.

File: pkg/runtime/wfengine/wfengine_test.go (hunk 6 above)
Comment (ItalyPaleAle) [reply pkg/runtime/wfengine/wfengine_test.go:786]: Or actually, `require.NoError` in this case. The difference is that `require` aborts the current test.

Comment (cgillum) [reply pkg/runtime/wfengine/wfengine_test.go:786]: > Or actually, `require.NoError` in this case. The difference is that `require` aborts the current test.

Oh wow, I didn't know about this. I have a lot of test code that I need to go back and clean up now. 😅

File: pkg/runtime/wfengine/wfengine.go (hunk 3 above)
Comment (mukundansundar) [reply pkg/runtime/wfengine/wfengine.go]: yes that is what I meant.

File: pkg/runtime/wfengine/wfengine.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go]: Found one place in sentry, where they had similar use of using `default`, if nothing found.
I have placed a common logic in utils.

File: pkg/runtime/wfengine/backend.go [hunk 7]
```diff
@@ -172,10 +182,14 @@ func (be *actorBackend) AddNewOrchestrationEvent(ctx context.Context, id api.Ins
 		return err
 	}
 
+	var actorType string
+	if be.config != nil {
+		actorType = be.config.WorkflowActorType
```
Comment (mukundansundar) [inline pkg/runtime/wfengine/backend.go:187]: is empty string a valid actor type? 
Should an error be returned if config is nil?

File: pkg/runtime/wfengine/wfengine.go [hunk 8]
```diff
@@ -46,37 +42,50 @@ type WorkflowEngine struct {
...
 
 	return engine
 }
 
 // InternalActors returns a map of internal actors that are used to implement workflows
 func (wfe *WorkflowEngine) InternalActors() map[string]actors.InternalActor {
 	internalActors := make(map[string]actors.InternalActor)
-	internalActors[WorkflowActorType] = wfe.workflowActor
-	internalActors[ActivityActorType] = wfe.activityActor
+	if wfe.config != nil {
```
Comment (mukundansundar) [inline pkg/runtime/wfengine/wfengine.go:85]: when can the config here be nil? 
Please add UTs for the same as necessary.

File: utils/utils.go [hunk 9]
```diff
@@ -198,3 +200,12 @@ func ComponentLogName(name, typ, version string) string {
 
 	return fmt.Sprintf(logNameVersionFmt, name, typ, version)
 }
+
+// GetNamespace returns the namespace for Dapr, or the default namespace if it is not set.
+func GetNamespaceOrDefault() string {
```
Comment (mukundansundar) [inline utils/utils.go:205]: Please add UTs for this.

File: pkg/runtime/wfengine/wfengine.go [hunk 10]
```diff
@@ -46,37 +42,50 @@ type WorkflowEngine struct {
...
+}
+
 func IsWorkflowRequest(path string) bool {
 	return backend.IsDurableTaskGrpcRequest(path)
 }
 
-func NewWorkflowEngine() *WorkflowEngine {
+func NewWorkflowEngine(config *WFConfig) *WorkflowEngine {
+	config.WorkflowActorType = actors.InternalActorTypePrefix + utils.GetNamespaceOrDefault() + "." + config.AppID + ".workflow"
+	config.ActivityActorType = actors.InternalActorTypePrefix + utils.GetNamespaceOrDefault() + "." + config.AppID + ".activity"
```
Comment (mukundansundar) [inline pkg/runtime/wfengine/wfengine.go:66]: have `.` as a separator and const and have workflow and activity as consts as well.

Comment (mukundansundar) [inline pkg/runtime/wfengine/wfengine.go:66]: This is going to be a breaking change. Is that expected?
cc @cgillum

**Review by mukundansundar — COMMENTED** [review]
Please add additional code paths after config nil checks, as in does it make sense to have actorType empty or ActivityActorType empty?

File: pkg/runtime/wfengine/backend.go (hunk 7 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/backend.go:187]: Was Doing that only, haven't yet requested re-review

**Comment by DeepanshuA** [comment]
> Please add additional code paths after config nil checks, as in does it make sense to have actorType empty or ActivityActorType empty?

Have been doing that only, haven't yet requested re-review

File: pkg/runtime/wfengine/wfengine.go (hunk 10 above)
Comment (cgillum) [reply pkg/runtime/wfengine/wfengine.go:66]: @mukundansundar yes, we're intentionally doing this as a breaking change as part of v1.11.

**Comment by cgillum** [comment]
> > Please reference the issue this PR will close: #6073
> 
> @DeepanshuA this is referring an issue that has been withdrawn and closed without work. What is the right issue to link to this PR?

@DeepanshuA please link this PR instead directly to https://github.com/dapr/dapr/issues/6013.

**Comment by DeepanshuA** [comment]
> > Please add additional code paths after config nil checks, as in does it make sense to have actorType empty or ActivityActorType empty?
> 
> Have been doing that only, haven't yet requested re-review :)

Alright, I am done with changes that I was intending to make. I will take care of rest of comments now. Thanks for the quick feedback.

File: pkg/runtime/wfengine/backend.go (hunk 7 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/backend.go]: I am done with the changes, hope this should be fine now

File: pkg/runtime/wfengine/wfengine.go (hunk 8 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go:85]: Ideally, config here won't be null as per the way current code is written, but I have placed a check accordingly.

File: pkg/runtime/wfengine/activity_test.go [hunk 11]
```diff
@@ -12,28 +12,31 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package wfengine_test
+package wfengine
```
Comment (cgillum) [inline pkg/runtime/wfengine/activity_test.go:15]: Is it necessary to change the package name from `wfengine_test` to `wfengine`? I was hoping to avoid this to keep the tests more blackbox friendly.

Comment (DeepanshuA) [reply pkg/runtime/wfengine/activity_test.go:15]: Actually, will have to expose `config` outside wfengine, for testing purpose then. And, didn't want to do so, just for testing. Hence, used same package as source code.
Is there something specific that we would be able to achieve with black-box testing? What I could understand that both strategies are fine (ref: https://stackoverflow.com/questions/19998250/proper-package-naming-for-testing-with-the-go-language).

File: pkg/runtime/wfengine/wfengine.go (hunk 8 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go:94]: Added UT.

File: utils/utils.go (hunk 9 above)
Comment (DeepanshuA) [reply utils/utils.go:212]: Added.

File: pkg/runtime/wfengine/wfengine.go (hunk 10 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go]: Please check now.

File: pkg/runtime/wfengine/wfengine.go (hunk 5 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go]: Please comment if this approach looks fine, or you think something better can be done to propagate this info.

**Comment by cgillum** [comment]
Hi @mukundansundar - I noticed you removed this from the v1.11 GitHub milestone. Are we really removing this from the v1.11 release?

**Comment by mukundansundar** [comment]
> Hi @mukundansundar - I noticed you removed this from the v1.11 GitHub milestone. Are we really removing this from the v1.11 release?

I added back the underlying issue into 1.11. Ideally we have either only the PR or the underlying issue tracked in the milestone.

File: pkg/runtime/wfengine/wfengine.go [hunk 12]
```diff
@@ -46,38 +43,61 @@ type WorkflowEngine struct {
...
 
-func NewWorkflowEngine() *WorkflowEngine {
+func NewWorkflowEngine(config *WFConfig) *WorkflowEngine {
+	actorTypePerAppPrefix := strings.Builder{}
+	actorTypePerAppPrefix.WriteString(actors.InternalActorTypePrefix)
+	actorTypePerAppPrefix.WriteString(utils.GetNamespaceOrDefault())
+	actorTypePerAppPrefix.WriteString(utils.DotDelimiter)
+	actorTypePerAppPrefix.WriteString(config.AppID)
+	actorTypePerAppPrefix.WriteString(utils.DotDelimiter)
+	actorTypePerAppPrefixStr := actorTypePerAppPrefix.String()
```
Comment (cgillum) [inline pkg/runtime/wfengine/wfengine.go:72]: This manual string-building seems like unnecessary complexity. I would expect something like this for extremely performance-sensitive code. However, this is *not* performance sensitive (we initialize the engine exactly once) so I find this kind of code to be distracting and unnecessarily hard to understand.

File: pkg/runtime/wfengine/wfengine.go [hunk 13]
```diff
@@ -46,38 +43,61 @@ type WorkflowEngine struct {
...
+
+	config.WorkflowActorType = actorTypePerAppPrefixStr + utils.WorkflowNameLabelKey
+	config.ActivityActorType = actorTypePerAppPrefixStr + utils.ActivityNameLabelKey
```
Comment (cgillum) [inline pkg/runtime/wfengine/wfengine.go:75]: It doesn't make sense to initialize the config object that's passed in as a constructor. I think of this as a bit of an anti-pattern since config objects should generally be immutable. It seems to me that you should simply pass compute the actor type value using the app ID and namespace and then just pass the actor type value into the `NewActivityActor` and `NewWorkflowActor` functions.

File: pkg/runtime/wfengine/activity_test.go (hunk 11 above)
Comment (cgillum) [reply pkg/runtime/wfengine/activity_test.go:15]: `WFConfig` is already a public type, so it's already exposed outside of `wfengine`. But regardless, I think you should instead have a `engine.GetActivityActorType()` and `engine.GetWorkflowActorType()` rather than `engine.config.XXXActorType`.

Generally speaking, I want to minimize the churn in this PR. I feel like it can be much smaller than it is right now.

File: pkg/runtime/wfengine/wfengine.go (hunk 13 above)
Comment (cgillum) [reply pkg/runtime/wfengine/wfengine.go:75]: Alternatively, consider initializing the workflow and activity actor types when constructing the `WFConfig` object (perhaps using a `NewWorkflowConfig` function) and exposing those types via read-only methods or fields.

**Comment by cgillum** [comment]
> I added back the underlying issue 6013 into 1.11. Ideally we have either only the PR or the underlying issue tracked in the milestone.

Got it - thanks for the clarification.

File: pkg/runtime/wfengine/wfengine.go (hunk 12 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go]: Removed

File: pkg/runtime/wfengine/wfengine.go (hunk 13 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go]: Changed as discussed

File: pkg/runtime/wfengine/activity_test.go (hunk 11 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/activity_test.go]: Changed as discussed

File: utils/utils.go [hunk 14]
```diff
@@ -26,6 +26,15 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+const (
+	defaultNamespace = "default"
+	DotDelimiter     = "."
+
+	// Workflow
+	WorkflowNameLabelKey = "workflow"
+	ActivityNameLabelKey = "activity"
```
Comment (cgillum) [inline utils/utils.go:35]: I think it's probably better to define the workflow and activity const values in the wfengine package since they're really only applicable in the context of the workflow engine.

**Review by cgillum — APPROVED** [review]
This looks great - I'm much happier with this newest iteration. Thanks for your patience with applying the feedback!

File: pkg/runtime/wfengine/activity_test.go [hunk 15]
```diff
@@ -30,10 +30,10 @@ import (
 func TestDedupeActivityInvocation(t *testing.T) {
 	// We do just enough to get the activity actor properly initialized
 	ctx := context.Background()
-	_, engine := startEngine(ctx, task.NewTaskRegistry())
+	_, engine := startEngine(ctx, t, task.NewTaskRegistry())
```
Comment (mukundansundar) [inline pkg/runtime/wfengine/activity_test.go:33]: every occurrence of `startEngine` seems to be preceded with a `context.Background` call ... Do you really need context to be passed in, can it just be init inside the `startEngine` function itself?

File: pkg/runtime/wfengine/wfengine.go [hunk 16]
```diff
@@ -46,37 +42,56 @@ type WorkflowEngine struct {
...
 
+// wfConfig is the configuration for the workflow engine
+type wfConfig struct {
+	AppID             string
+	workflowActorType string
+	activityActorType string
+}
+
+// NewWorkflowConfig creates a new workflow engine configuration
+func NewWorkflowConfig(appID string) *wfConfig {
```
Comment (mukundansundar) [inline pkg/runtime/wfengine/wfengine.go:61]: why is a pointer to this struct returned? Just a normal struct is enough here?
Am I correct in saying that the value is written once and accessed multiple times, so in that case why is a pointer needed here?

File: utils/utils_test.go [hunk 17]
```diff
@@ -270,3 +270,16 @@ func TestMapToSlice(t *testing.T) {
...
 }
+
+func TestGetNamespaceOrDefault(t *testing.T) {
+	t.Run("namespace is empty", func(t *testing.T) {
+		ns := GetNamespaceOrDefault()
+		assert.Equal(t, "default", ns)
+	})
+
+	t.Run("namespace is not empty", func(t *testing.T) {
+		os.Setenv("NAMESPACE", "testNs")
```
Comment (mukundansundar) [inline utils/utils_test.go:281]: use `t.Setenv` instead

File: pkg/runtime/wfengine/activity_test.go (hunk 15 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/activity_test.go:33]: Can do so but let's take a case like `TestEmptyWorkflow`: https://github.com/dapr/dapr/blob/3fb850d12c5306f1dd424cfa15b41d27f0c4e0e7/pkg/runtime/wfengine/wfengine_test.go#L107

There, we also need to pass `context` to `ScheduleNewOrchestration` and `WaitForOrchestrationCompletion`, and if I understand correctly, a Prod scenario will have same context being passed in there after starting Engine and then to schedule and wait for an orchestration. So, current test mimics almost similar scenario with same context being passed to them.
The tests would ideally also work with `context.TODO()` in all these places. But, I feel it is better to use a common context, which is being done currently.
WDYT?

File: pkg/runtime/wfengine/wfengine.go (hunk 16 above)
Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine.go:61]: Nice point. It doesn't need to be mutable. And, with a pointer, it would also need to do `escape analysis` and it might also land up in heap, where it is not needed actually. I will change this one.

File: pkg/runtime/wfengine/wfengine_test.go [hunk 18]
```diff
@@ -600,8 +609,8 @@ func startEngine(ctx context.Context, r *task.TaskRegistry) (backend.TaskHubClie
 	return client, engine
```
Comment (mukundansundar) [inline pkg/runtime/wfengine/wfengine_test.go:609]: remove panic in line 607.

File: pkg/runtime/wfengine/wfengine_test.go [hunk 19]
```diff
@@ -587,9 +596,9 @@ func TestPauseResumeWorkflow(t *testing.T) {
 	}
 }
 
-func startEngine(ctx context.Context, r *task.TaskRegistry) (backend.TaskHubClient, *wfengine.WorkflowEngine) {
+func startEngine(ctx context.Context, t *testing.T, r *task.TaskRegistry) (backend.TaskHubClient, *wfengine.WorkflowEngine) {
```
Comment (mukundansundar) [inline pkg/runtime/wfengine/wfengine_test.go:599]: is there a proper way to stop engine? If so please add code for that too. 
Are there race conditions in these tests anywhere? If not add them to the list of packages that do not have race conditions.

Comment (mukundansundar) [reply pkg/runtime/wfengine/wfengine_test.go:599]: if this requires more work create an issue and create a separate PR for the same.

Comment (DeepanshuA) [reply pkg/runtime/wfengine/wfengine_test.go:599]: I have created https://github.com/dapr/dapr/issues/6264 as it is a wider issue, separate from current PR.

File: pkg/sentry/certs/store.go [hunk 20]
```diff
@@ -12,10 +12,7 @@ import (
 	"github.com/dapr/dapr/pkg/sentry/config"
 	"github.com/dapr/dapr/pkg/sentry/consts"
 	"github.com/dapr/dapr/pkg/sentry/kubernetes"
-)
-
-const (
-	defaultSecretNamespace = "default"
+	"github.com/dapr/dapr/utils"
```
Comment (mukundansundar) [inline pkg/sentry/certs/store.go:15]: this package is already used in `sentry/config/config.go`. It's not newly added to this binary. 
https://github.com/dapr/dapr/blob/master/pkg/sentry/config/config.go#L15

File: pkg/sentry/certs/store.go [hunk 21]
```diff
@@ -12,10 +12,7 @@ import (
 	"github.com/dapr/dapr/pkg/sentry/config"
 	"github.com/dapr/dapr/pkg/sentry/consts"
 	"github.com/dapr/dapr/pkg/sentry/kubernetes"
-)
-
-const (
```
Comment (mukundansundar) [inline pkg/sentry/certs/store.go:17]: @DeepanshuA Please revert the changes to this file alone. 
Have the utils function ... but use the local var and local function defined in this file and not the utils one.

**Review by mukundansundar — CHANGES_REQUESTED** [review]

File: pkg/sentry/certs/store.go (hunk 21 above)
Comment (DeepanshuA) [reply pkg/sentry/certs/store.go:17]: Accepting a param as defaultNamespace now in utils.

File: pkg/sentry/certs/store.go [hunk 22]
```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/dapr/dapr/pkg/sentry/config"
 	"github.com/dapr/dapr/pkg/sentry/consts"
 	"github.com/dapr/dapr/pkg/sentry/kubernetes"
+	"github.com/dapr/dapr/utils"
```
Comment (mukundansundar) [inline pkg/sentry/certs/store.go:15]: utils package is already used in `sentry/config/config.go`

**Review by mukundansundar — APPROVED** [review]
